autopilot/combinedattach.go

```diff
@@ -126,3 +127,23 @@ func (c *WeightedCombAttachment) NodeScores(g ChannelGraph, chans []Channel,
 
 	return scores, nil
 }
+
+// SetNodeScores is used to set the internal map from NodeIDs to scores if a
+// score settable sub-heuristic is active.
+//
+// NOTE: This is a part of the ScoreSettable interface.
+func (c *WeightedCombAttachment) SetNodeScores(
+	newScores map[NodeID]float64) error {
```

Comment:
Seems like we should instead make it a bit more precise to _only_ target the specified heuristic. Otherwise, in the future we may accidentally override a set of heuristics inadvertently.

Reply:
True. I prefer deferring this to when we actually have more heuristics to target though, as I'm not sure what is the proper way of targetting this as long as we have only one.

Reply:
Could use an enum for each heuristic, but agree that can be deferred until needed.